connect/eaas/core/validation/validators/eventsapp.py

```diff
@@ -17,6 +17,7 @@
 def validate_eventsapp(context):
 
     extension_class = context['extension_classes'].get('extension')
+    extension_class = extension_class or context['extension_classes'].get('eventsapp')
```

Comment:
If not wrong, we need to look for events app to now....

Reply:
Good catch thanks @bdjilka !